src/xunit.console.uwp/HostedAppxTest.cs

```diff
@@ -162,19 +163,20 @@ public void Run(bool debug, TimeSpan timeout)
             }
             else
             {
-                Console.WriteLine("No results found at  {0}, copying logs ", resultPath);
```

Comment:
We should still copy them if they exist. (Essentially, move the "copy logs" out of the else of File.Exists(resultPath))